defaults/dark-theme.toml

```diff
@@ -61,6 +61,11 @@ yellow = "#E5C07B"
 "variable.other.member" = "$green"
 "tag" = "$blue"
 
+"bracket.color.1" = "$blue"
+"bracket.color.2" = "$yellow"
+"bracket.color.3" = "$purple"
+"bracket.unpaired" = "$red"
+
```

Comment:
This is still missing from config file

Reply:
The parser lives in core so it just returns String like tree-sitter. There is no need to access via LapceTheme config.

Reply:
That doesn't matter though? It still has to be part of the config

Reply:
Do you mean lapce-data/config.rs? Other syntax colors are also not included though?

Reply:
Now that I'm looking at it, it's correct, my bad